frontend/src/app/model/selections/SliceSelection.ts

```diff
@@ -19,4 +19,17 @@ export abstract class SliceSelection {
     }
 
     public abstract toJSON(): Object;
+    public getAdditionalData(): Object {
+        return {};
+    }
+
+    protected static base64toBlob(base64: string, dataType: string = 'image/png'): Blob {
```

Comment:
Is there a better place for such functions?

Reply:
Well, some service or util will be good. There are few functions that can be then moved there.